example/swagger-files/parameter-styles-cookie.json

```diff
@@ -0,0 +1,131 @@
+{
+    "openapi": "3.0.0",
+    "info": {
+      "description": "This oas file demonstrates all of the different cookie parameter styles",
+      "version": "1.0.0",
+      "title": "Parameter Styles",
+      "contact": {
+        "email": "aaron@readme.io"
+      },
+      "license": {
+        "name": "Apache 2.0",
+        "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+      }
+    },
+    "paths": {
+        "/nonExplodedFormCookies": {
+            "get": {
+                "summary": "Demonstrate all non-exploded form cookie styles",
+                "operationId": "demonstrateCookies",
+                "parameters": [{
+                    "name": "primitive",
+                    "in": "cookie",
+                    "description": "Form style primitive",
+                    "required": true,
+                    "style": "form",
+                    "explode": false,
+                    "schema": {
+                        "type": "string"
+                    }
+                }, {
+                    "name": "array",
+                    "in": "cookie",
+                    "description": "Form style array. Note: this is encoded in https://github.com/readmeio/httpsnippet/blob/master/src/index.js#L94, it's unclear whether this encoding is the correct behavior because the cookie header deviates from all other headers.",
+                    "required": true,
+                    "style": "form",
+                    "explode": false,
+                    "schema": {
+                        "type": "array",
+                        "items": {
+                            "type": "string"
+                        }
+                    }
+                }, {
+                    "name": "object",
+                    "in": "cookie",
+                    "description": "Form style object. Note: this is encoded in https://github.com/readmeio/httpsnippet/blob/master/src/index.js#L94, it's unclear whether this encoding is the correct behavior because the cookie header deviates from all other headers.",
+                    "required": true,
+                    "style": "form",
+                    "explode": false,
+                    "schema": {
+                        "type": "object",
+                        "properties": {
+                            "name": {
+                                "type": "string"
+                            },
+                            "description": {
+                                "type": "string"
+                            }
+                        }
+                    }
+                }],
+                "responses": {
+                    "200": {
+                        "description": "Successful response"
+                    }
+                }
+            }
+        },
+        "/explodedFormCookies": {
+            "get": {
+                "summary": "Demonstrate all exploded form cookie styles",
```

Comment:
Is it right that properties within `object` get hoisted up when exploded?

![Screen Shot 2021-01-06 at 10 31 34 AM](https://user-images.githubusercontent.com/33762/103806851-75222880-500a-11eb-8a29-0e232fe7b8f4.png)

Reply:
Correct, that's basically the definition of explode. Turn each key=>value pair into a top level key. There are some edge cases that don't respect this though, and I've called it out in the tests.